Libraries/test/Amazon.Lambda.Annotations.SourceGenerators.Tests/SourceGeneratorTests.cs

```diff
@@ -284,7 +284,6 @@ public async Task VerifyFunctionInSubNamespace()
         {
             var expectedTemplateContent = (await File.ReadAllTextAsync(Path.Combine("Snapshots", "ServerlessTemplates", "subnamespace.template"))).ToEnvironmentLineEndings();
             var expectedSubNamespaceGenerated = (await File.ReadAllTextAsync(Path.Combine("Snapshots", "Functions_ToUpper_Generated.g.cs"))).ToEnvironmentLineEndings();
-            var expectedProgramGenerated = (await File.ReadAllTextAsync(Path.Combine("Snapshots", "Program.g.cs"))).ToEnvironmentLineEndings();
```

Comment:
Why remove this (and the next one)? Were we missing the assert against it?

Reply:
The `Program.g.cs` does not even get generated since the source files being supplied to  the test were from the `TestServerlessApp` and not the `TestExecutableServerlessApp`. 

We weren't asserting against `expectedProgramGenerated` and hence removed it from both places.